.pre-commit-config.yaml

```diff
@@ -20,8 +20,8 @@ repos:
         files: \.(md|rst|toml|yml|yaml)
         args: [--prose-wrap=preserve]
 
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: 6fcbf19c7cf351a8dc3517a0fed3ebfa05610de2 # frozen: v0.0.290
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: d7c4946c7c152163ce34efe35be4e1015d053edd # frozen: v0.1.8
```

Comment:
Is there a reason not to use 0.1.9?

Reply:
Absolutely not. I thought you might want to avoid the bleeding edge.